docs/console/company-configuration/providers/extension-providers.md

```diff
@@ -0,0 +1,40 @@
+---
+id: extension-providers
+title: Extension Providers
+sidebar_label: Extension Providers
+---
+
+:::info
+
+Extension Providers can only be used when the [Enhanced Project Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) is enabled.
+
+:::
+
+Some providers can extend or replace specific Console features.
+
+The steps to enable a Project Extension, using an Extension Provider, are the following:
+
+- check that the [Enhanced Project Workflow](/development_suite/set-up-infrastructure/enhanced-project-workflow.md) is enabled for the project; 
+- connect a _capable Provider_ (more details later) to its Company, as described in the [Management page](/console/company-configuration/providers/configure-provider.mdx);
```

Comment:
capable of what? 🤔 

also here you are explaining how to setup a provider by telling to setup the provider.

moreover nothing says that the capabilities provided by an Extension API provider are only meant for projects.


In this page I'd explain the purpose of an Extension API Provider, where to create it and the supported capabilities.

Then in the specific page for each capability explain how to activate it according to how it works relatively on the feature it affects; for instance we could create a "Cluster Provisioner" capability to the Extension API provider, such capabilities would only mke sense within the Company and make no sense in the project settings

Reply:
I moved the steps in the Orchestrator Generator page, added a general description of an Extension and a screenshot of the modal adding an Extension API Provider